src/components/Settings/SettingsJobsCache/index.tsx

```diff
@@ -126,6 +152,111 @@ const SettingsJobs: React.FC = () => {
           intl.formatMessage(globalMessages.settings),
         ]}
       />
+      <Transition
+        enter="opacity-0 transition duration-300"
+        enterFrom="opacity-0"
+        enterTo="opacity-100"
+        leave="opacity-100 transition duration-300"
+        leaveFrom="opacity-100"
+        leaveTo="opacity-0"
+        show={jobEditModal.isOpen}
+      >
+        <Formik
+          initialValues={{
+            jobSchedule: '',
+          }}
+          validationSchema={Yup.object().shape({
+            jobSchedule: Yup.string().required(),
+          })}
+          onSubmit={async (values) => {
+            try {
+              setIsSaving(true);
+              await axios.post(
+                `/api/v1/settings/jobs/${jobEditModal.job?.id}/schedule`,
+                {
+                  ids: jobEditModal.job?.id,
+                  schedule: values.jobSchedule,
+                }
+              );
+              addToast(intl.formatMessage(messages.jobScheduleEditSaved), {
+                appearance: 'success',
+                autoDismiss: true,
+              });
+              setJobEditModal({ isOpen: false });
+            } catch (e) {
+              addToast(intl.formatMessage(messages.jobScheduleEditFailed), {
+                appearance: 'error',
+                autoDismiss: true,
+              });
+            } finally {
+              setIsSaving(false);
+            }
+          }}
+        >
+          {({ handleSubmit, isSubmitting, errors, touched }) => {
+            return (
+              <Modal
+                title={intl.formatMessage(messages.editJobSchedule)}
+                okText={
+                  isSubmitting
+                    ? intl.formatMessage(globalMessages.saving)
+                    : intl.formatMessage(globalMessages.save)
+                }
+                iconSvg={<SolidPencilIcon className="w-5 h-5" />}
```

Comment:
```suggestion
                iconSvg={<SolidPencilIcon className="w-6 h-6" />}
```
(@sct this is why we need #1464 😉)